### DISCUSSION TITLE
Windows - Rustdesk event id 4798 "A user s local group membership was enumerated"

### DISCUSSION THREAD

**Discussion by Git-ChrisSc** [rustdesk/rustdesk#7071] [General]
### Bug Description

Every 2 Minutes Rustdesk does enumerated local group Membership.
Multiple local Useraccounts (Administrator/Guest/etc.)

### How to Reproduce

Just installed on Windows

### Expected Behavior

I will know, why its happens?
Why every 2 Minutes?
Why is this needed?

### Operating system(s) on local side and remote side

Windows 10 + Windows 11

### RustDesk Version(s) on local side and remote side

Client 1.2.3 

### Screenshots

![grafik](https://github.com/rustdesk/rustdesk/assets/76469940/98f25313-00f4-47d9-a43e-19e14e8861b5)

### Additional Context

Hey everyone,

Thank you for this nice Project! 
Today i see many Events in the Logs that i describe upwards. I checked it on 2 PCs an both show the same behavior.

Can this behavior explained, 
and can the interval changed?

**Comment by rustdesk** [comment]
Thanks for your finding. 
Appreciate if you can figure it out and improve, https://github.com/rustdesk/rustdesk/blob/master/src/platform/windows.rs

**Reply by Git-ChrisSc** [reply]
Thank you for replay.

But, excuse me, I have not the knowledge to improve the code or find something where and why it happens. I never have learned "rust" or programming

**Comment by pkess** [comment]
I had a short Look into the Code. Maybe the Function `get_sid_of_user` causes this log entry:
https://github.com/rustdesk/rustdesk/blob/master/src/platform/windows.rs#L2431

**Reply by rustdesk** [reply]
But get_sid_of_user is not called every 2 minutes.

**Comment by Git-ChrisSc** [comment]
@rustdesk did you see here a possibility - that the get_sid_of_user Fkt throw the event-ids?
Is this topic realy a discussion or is this maybe an issue?

**Reply by rustdesk** [reply]
If you have any way to avoid this in get_sid_of_user, please advise.
